mypy/build.py

```diff
@@ -1994,8 +2003,14 @@ def parse_file(self) -> None:
             return
 
         manager = self.manager
+
+        # Can we reuse a previously parsed AST? This avoids redundant work in daemon.
+        cached = self.id in manager.ast_cache and True
```

Comment:
Why do you need `and True`? Leftover from debugging?

Reply:
Yeah, I'll remove it.